1/src/_backend/schema.js

```diff
@@ -54,6 +59,9 @@ type JWebtoken {
     ASC
     DESC
 }
-  
+
 `;
-module.exports = typeDefs;
\ No newline at end of file
+module.exports  = makeAugmentedSchema({
+  typeDefs: typeDefs
+  //resolvers,
```

Comment:
If you're not using your own custom resolvers, you should remove that file, ie. remove dead code.